test/e2e/fleetautoscaler_test.go

```diff
@@ -416,7 +416,17 @@ func TestTlsWebhook(t *testing.T) {
 	}
 
 	// since we're using statically-named service, perform a best-effort delete of a previous service
-	framework.KubeClient.CoreV1().Services(defaultNs).Delete(svc.ObjectMeta.Name, waitForDeletion) // nolint:errcheck
+	err = framework.KubeClient.CoreV1().Services(defaultNs).Delete(svc.ObjectMeta.Name, waitForDeletion)
```

Comment:
needs DeepCopy() here to avoid data race

Reply:
otherwise LGTM

Reply:
:+1: deepcopying all the things.